apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryHandler.cs

```diff
@@ -37,30 +37,35 @@ internal sealed class RetryHandler : IHttpUnsuccessfulResponseHandler
         internal const string InvocationIdHeaderPart = "gccl-invocation-id";
         internal const string AttemptCountHeaderPart = "gccl-attempt-count";
 
-        private static readonly int[] s_retriableErrorCodes =
+        private double _backoffMultiplier;
```

Comment:
Why is `_backoffMultiplier` not `readonly`?

Reply:
I am not sure if we should do it, but we could have just created two readonly fields `_retryTimings` and `_retryPredicate` and everything else would have just been taken from these two fields.